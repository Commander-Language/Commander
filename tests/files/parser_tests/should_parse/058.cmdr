// Test that lambda expressions without curly braces parses correctly.
addOne = (x,y) -> x+1;
