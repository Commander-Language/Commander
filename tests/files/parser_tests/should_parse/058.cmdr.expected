(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode addOne) SET (LambdaExprNode (BindingsNode (BindingNode x) (BindingNode y)) (ReturnStmtNode (BinOpExprNode (LValueExprNode (VarExprNode x)) ADD (IntExprNode 1)))))))
